gcc/config/riscv/zce.md

```diff
@@ -22,7 +22,7 @@
 (define_insn "*zero_extendsidi2_zcee"
   [(set (match_operand:DI 0 "register_operand" "=r,r")
 	(zero_extend:DI (match_operand:SI 1 "nonimmediate_operand" "r,m")))]
-  "TARGET_64BIT && TARGET_ZCEE"
+  "TARGET_64BIT && TARGET_ZCEE && TARGET_ZEXT"
```

Comment:
If we just use option -march=zcee, the instruction may not gen, so I think these better use || but not && TARGET_ZEXT

Reply:
I think we must always use mzce-sext and mzce-zext to start sext.* and zext.*. 
If we just only use -march=zcee,the option can not transmit to binutils.
Without mzce-sext in binutils , sext.* zext.* only can not be used .